src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -96,4 +106,24 @@ public double getRightWheelsRPM() {
   public double getAverageWheelsRPM() {
     return (getLeftWheelsRPM() + getRightWheelsRPM()) / 2;
   }
+
+  @Override
+  public double getMeasurement(){
+    return getDistance();
+  }
+
+  @Override
+  public void useOutput(double output, double setpoint){
+    m_leftMotors.setVoltage(output + m_driveTrainFeedforward.calculate(setpoint));
+    m_rightMotors.setVoltage(output + m_driveTrainFeedforward.calculate(setpoint));
+  }
+
+  public boolean atSetpoint(){
+    return getController().atSetpoint();
+  }
+
+  @Override
+  public void periodic(){
+    setSetpoint(DriveTrainConstants.kInitialSetPoint - getMeasurement());
```

Comment:
Remove this line, you do not want to change the setpoint every 20ms... You want it to be set by the autonomous command that will run.